phpBB/phpbb/cache/service.php

```diff
@@ -141,6 +141,7 @@ function obtain_icons()
 				$icons[$row['icons_id']]['img'] = $row['icons_url'];
 				$icons[$row['icons_id']]['width'] = (int) $row['icons_width'];
 				$icons[$row['icons_id']]['height'] = (int) $row['icons_height'];
+				$icons[$row['icons_id']]['alt'] = ($row['icons_alt']) ? $row['icons_alt'] : $row['icons_url'];
```

Comment:
I'd rather have this empty than the icons URL. Nobody cares about the URL if there is no alternative text set.